rollup.config.js

```diff
@@ -118,23 +128,34 @@ if (!buildBundles) {
     },
   ];
 } else {
-  rollupConfigs = [
-    {
-      input: libInput,
-      output: [
-        {
-          // hack to have the mxGraph configuration prior the load of the mxGraph lib
-          banner: readFileSync('src/static/js/configureMxGraphGlobals.js') + '\n' + readFileSync('node_modules/mxgraph/javascript/mxClient.min.js'),
-          file: pkg.browser,
-          name: 'bpmnvisu',
-          format: 'iife',
-        },
-      ],
-      // TODO we may use this plugin configuration instead resolve({browser: true})
-      // If true, instructs the plugin to use the "browser" property in package.json files to specify alternative files to load for bundling. This is useful when bundling for a browser environment.
-      plugins: plugins,
-    },
-  ];
+  const configIife = {
+    input: libInput,
+    output: [
+      {
+        // hack to have the mxGraph configuration prior the load of the mxGraph lib
+        banner: readFileSync('src/static/js/configureMxGraphGlobals.js') + '\n' + readFileSync('node_modules/mxgraph/javascript/mxClient.min.js'),
+        file: pkg.browser,
+        name: 'bpmnvisu',
+        format: 'iife',
+      },
+    ],
+    // TODO we may use this plugin configuration instead resolve({browser: true})
+    // If true, instructs the plugin to use the "browser" property in package.json files to specify alternative files to load for bundling. This is useful when bundling for a browser environment.
+    plugins: plugins,
+  };
+  const configEsmMin = {
+    input: libInput,
+    output: [
+      {
+        file: pkg.module.replace('.js', '.min.js'),
+        format: 'es',
+      },
+    ],
+    external: ['entities/lib/decode', 'fast-xml-parser/src/parser'],
+    plugins: pluginsNoDeps,
+  };
+  const configEsm = { ...configEsmMin, plugins: pluginsNoDepsNoMin, output: { file: pkg.module, format: 'es' } };
```

Comment:
:question: I don't remember we need a minified js for this. Let's review this together

Reply:
We decided to provide the 2 files

Reply:
as discussed on meet ok to have the min version